google-map.js

```diff
@@ -430,6 +431,26 @@ Polymer({
     singleInfoWindow: {
       type: Boolean,
       value: false
+    },
+
+    /**
+     * Secret sauce Google gave to us.
+     */
+    fleetEngineAccessToken: {
+      type: String,
+      value: null,
+      observer: '_fleetEngineAccessTokenChanged'
+    },
+
+    fleetEngineProjectId: {
+      type: String,
+      value: ""
+    },
+
+    deliveryVehicleId: {
+      type: String,
+      value: "",
+      observer: '_deliveryVehicleChanged'
```

Comment:
avoid mixing single and double quoted strings (running prettier should be fine)

Reply:
Can't use prettier, this is a fork and prettier is not used upstream...